bin/plugin/cli.js

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env node
+
+// Dependencies
+const program = require( 'commander' );
+const { releaseRC, releaseStable } = require( './commands/release' );
+const { prepublishNpmStablePackages } = require( './commands/packages' );
```

Comment:
```suggestion
/*
 * External dependencies
 */
const program = require( 'commander' );

/*
 * Internal dependencies
 */
const { releaseRC, releaseStable } = require( './commands/release' );
const { prepublishNpmStablePackages } = require( './commands/packages' );
```

Reply:
How did you defeat `@wordpress/eslint-plugin/dependency-group` 🤔

Reply:
It doesn't error for CommonJS :)

Reply:
> It doesn't error for CommonJS :)

I definitely had it in consideration for the implementation 🤔 

https://github.com/WordPress/gutenberg/blob/b4eba9a8e7e571d835da66e6372655b4e5af9369/packages/eslint-plugin/rules/dependency-group.js#L170-L180

Reply:
I assumed it only covers for ES Modules. It's been working like that for as long as I remember.